### PR TITLE
Added proxy support

### PR DESCRIPTION
**Pull request by markfalk**
This allows us to use this within an environment requiring proxy support.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
-            return callback(err);
-        } else if (resp.ok) {
-            return callback(null, resp.text);
-        }
-        callback(new Error('GET ' + remoteUrl + ' ' + resp.status));
-    });
+    var proxy_url = getProxyForUrl(remoteUrl);
+
+    if(proxy_url) {
+        superagent.get(remoteUrl).proxy(proxy_url).buffer().end(function (err, resp) {
```
Comment (jonkemp) [inline index.js]: Looks like the callback function is the same in both cases. This is duplicated code that could be factored out.

Also, please respect coding conventions and/or run eslint. Thanks.

**Comment by jonkemp** [comment]
I left some comments for your review.

**Review by jonkemp — CHANGES_REQUESTED** [review]

**Review by jonkemp — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "remote-content",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (jonkemp) [inline package.json]: Oh yeah, I just noticed this. Don't bump the package number in a pull request. I'll take care of it when it's published.

Comment (markfalk) [reply package.json]: Apologies for that.  I've submitted another commit.

Comment (jonkemp) [reply package.json]: Would the proxy support be considered a new feature or a patch?

**Comment by jonkemp** [comment]
Would the proxy support be considered a new feature or a patch?

**Comment by markfalk** [comment]
I’d consider it a patch. It ensures that it will work in an environment which requires proxies. There are other libraries which do this by default. It broke our workflow until I identified the issue.

**Pull request merged**